metricbeat/docs/autodiscover-hints.asciidoc

```diff
@@ -1,4 +1,45 @@
-Metricbeat supports a autodiscover based on hints from the provider:
+Metricbeat supports a autodiscover based on hints from the provider. The `hints` system looks for
```

Comment:
Please apply my comments from the Filebeat topic to wherever they apply in this topic. I won't repeat my feedback here because the language is pretty similar.